src/Controller/EnMarche/UserController.php

```diff
@@ -41,8 +41,11 @@ public function profileOverviewAction(): Response
      */
     public function profileDonationAction(DonationRepository $donationRepository): Response
     {
+        $userEmail = $this->getUser()->getEmailAddress();
+
         return $this->render('user/my_donation.html.twig', [
-            'donations' => $donationRepository->findByEmailAddressOrderedByDonatedAt($this->getUser()->getEmailAddress()),
+            'donations' => $donationRepository->findByEmailAddressOrderedByDonatedAt($userEmail, 'DESC'),
+            'subscribed_donations' => $donationRepository->findAllSubscribedDonationByEmail($userEmail),
```

Comment:
1) why `findAll` if juste one subscription is possible?
2) `subscribed_donations` => `subscribed_donation`
3) do we really need to make 2 queries in DB for the same table?

Reply:
it could have multiple subscription, actualy it's not blocked

I use the same method to get subscribed donation that is used in cancel controller.